### PR TITLE
added a link to DAPT in profiles of TTML2 description

### PR DESCRIPTION
**Pull request by himorin**
@nigelmegitt @gkatsev how about like this?

***
<a href="https://pr-preview.s3.amazonaws.com/himorin/charter-timed-text/pull/73.html" title="Last updated on Feb 21, 2022, 2:23 PM UTC (e0a6020)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/charter-timed-text/73/fd859b3...himorin:e0a6020.html" title="Last updated on Feb 21, 2022, 2:23 PM UTC (e0a6020)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,7 +178,9 @@ <h3>New Normative Specifications</h3>
         for example by setting performance or complexity constraints.
       </p>
       <p>An example application would be the authoring and exchange of
-        timed text documents that support audio description or dubbing workflows. Such an application might define text associated with media at
+        timed text documents that support audio description or dubbing workflows, 
+        under development as the [Dubbing and Audio description Profiles of TTML2](https://github.com/w3c/dapt) specification.
```
Comment (nigelmegitt) [inline index.html]: Should this be an `<a>` instead of a Markdown-style link?

Comment (himorin) [reply index.html]: YES! thank you for pointing this.

Comment (himorin) [inline index.html]:
```suggestion
        under development as the <a href="https://github.com/w3c/dapt">Dubbing and Audio description Profiles of TTML2</a> specification.
```

**Review by nigelmegitt — APPROVED** [review]
LGTM

**Pull request merged**